### ISSUE TITLE
Typescript 4.0.5 - inconsistent property/accessor in ContainerWindow vs. DefaultContainerWindow

### ISSUE THREAD

**Issue by Sly1024** [Morgan-Stanley/desktopJS#385]
Compiling with Typescript 4.0.5 I received the following error messages:
```
 Error: node_modules/@morgan-stanley/desktopjs/dist/desktopjs.d.ts:677:21 - error TS2611: 'id' is defined as a property in class 'ContainerWindow', but is overridden here in 'DefaultContainerWindow' as an accessor.
    
    677                 get id(): string;
                            ~~
    node_modules/@morgan-stanley/desktopjs/dist/desktopjs.d.ts:678:21 - error TS2611: 'name' is defined as a property in class 'ContainerWindow', but is overridden here in 'DefaultContainerWindow' as an accessor.
    
    678                 get name(): string;
                            ~~~~
```

**Comment by Brookke** [comment]
I believe https://github.com/Morgan-Stanley/desktopJS/blob/2ec0c82494f8b926e66a46d6d67c5b1555741d21/packages/desktopjs/src/window.ts#L87 should fix this when it gets merged in #370 :)

**Comment by Sly1024** [comment]
Indeed, it fixes the issue. I tried it by manually editing the .d.ts file before the compilation.
I was going to do a PR and change these to property getters, but this should do the trick. 
When do you expect the merge to happen?

**Comment by Brookke** [comment]
I was waiting on some changes for #383 to avoid creating any issues with merge conflicts etc and also need to find some time in my sprint to take a look at the remaining changes.

I shall instead spin that change off into its own PR so that the fix can get in sooner

**Comment by Brookke** [comment]
PR open @Sly1024
